habushu-maven-plugin/src/test/resources/containerize-dependencies/default-single-monorepo-dep/test-monorepo/extensions/extensions-python-dep-X/pom.xml

```diff
@@ -15,18 +15,13 @@
             <plugin>
                 <groupId>org.technologybrewery.habushu</groupId>
                 <artifactId>habushu-maven-plugin</artifactId>
-                <version>2.15.0</version>
+                <version>${project.version}</version>
```

Comment:
Q: Can we remove the version entirely? Doesn't really make sense to set it to 1.0.0 either.

Reply:
yes, just tested, it works without the version